stable/AudibleCharacterStatus/manifest.toml

```diff
@@ -0,0 +1,10 @@
+[plugin]
+repository = "https://github.com/Arrenton/AudibleCharacterStatus.git"
+commit = "d4615cb10e8c6960f29609b595f219e7374245c9"
+owners = [
+    "Arrenton",
+]
+project_path = "AudibleCharacterStatus"
```

Comment:
Remove this line, your .csproj is in the root, this is only needed if the project folder is nested.